File: Java/_167_Two_Sum_II/Solution.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int start = 0;
        int end = numbers.length - 1;
        int[] result = new int[2];
        if (end < 0)
            return result;
        while (numbers[start] + numbers[end] != target && start < end) {           
            if (numbers[start] + numbers[end] < target)
                start++;
            else
                end--;
        }
        if (start > end)
            return result;
        else {
            result[0] = (start + 1);
            result[1] = (end + 1);
            return result;
        }
    }
}

File: Python/_167_Two_Sum_II/solution.py
class Solution:
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        start = 0;
        end = len(numbers) - 1
        result = []
        if end < 0:
            return result
        while numbers[start] + numbers[end] != target and start < end:
            if numbers[start] + numbers[end] < target:
                start += 1
            else:
                end -= 1
        if not start < end:
            return result
        else:
            result.append(start + 1)
            result.append(end + 1)
            return result